app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenControllerTest.kt

```diff
@@ -76,20 +66,18 @@ class PatientEditScreenControllerTest {
 
   private lateinit var screen: PatientEditScreen
   private lateinit var patientRepository: PatientRepository
-  private val numberValidator: PhoneNumberValidator = IndianPhoneNumberValidator()
   private lateinit var controller: PatientEditScreenController
 
-  private lateinit var errorConsumer: (Throwable) -> Unit
-  private lateinit var dobValidator: UserInputDateValidator
   private val dateOfBirthFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy", Locale.ENGLISH)
+  private val dobValidator: UserInputDateValidator = UserInputDateValidator(TestUserClock().zone, dateOfBirthFormat)
```

Comment:
We can just use `ZoneOffset.UTC` here since that is the default zone of the `TestUserClock`.